rts/at/bridge/src/main/java/org/jboss/narayana/rest/bridge/inbound/InboundBridge.java

```diff
@@ -212,11 +222,15 @@ private void enlist(final InboundBridge inboundBridge) {
      *
      * @return
      */
-    private Transaction getTransaction() {
+    private Transaction getTransaction(boolean useContainerToLookup) {
```

Comment:
I was wondering about that myself. I wonder if we can revert that now and always use the container lookup'er